lib/install/config/webpack/paths.yml

```diff
@@ -0,0 +1,7 @@
+# Restart webpack-watcher or webpack-dev-server if you make changes here
+paths:
+  src_path: app/javascript
+  config_path: config/webpack
+  node_modules_path: node_modules
+  dist_dir: packs
+  dist_path: public/packs
```

Comment:
`dist_dir` doesn't feel like the right name to me. It's the directory name for `src` entry files, not `dist` output files. Its usage in `shared.js` highlights the disconnect: `glob.sync(path.join(paths.src_path, paths.dist_dir, extensionGlob))`.

I propose:
```yaml
paths:
  config_path: config/webpack
  src_path: app/javascript/packs
  dist_path: public/packs
  node_modules_path: node_modules
  resolve_paths:
    - app/javascript
```

/cc @dhh